### ISSUE TITLE
Won't install for play.

### ISSUE THREAD

**Issue by almightycricket** [aa2g/AA2Unlimited#116]
It won't overwrite the play exe due to a size mismatch. I used the hongfire patch and I believe this screwed me. Is there a fix or is this intended?

**Comment by geneishouko** [comment]
Only the original exe is supported so you should get that and patch it

**Comment by apo555** [comment]
Hello, I have the original .exe file, but the patch still doesn't want to be installed. "aa2play.exe doesn't appear to be AA2play/edit exe file. rdata.size = 0x0, but expected around 0x71482". The patch works only for AA2 edit.exe.
The original file is from this tutorial and the disks are mounted with Daemon tools Lite :https://www.youtube.com/watch?v=vwOds8z2wwE ,where did I do something wrong ?

**Comment by Reinyn1** [comment]
Yes, i have a question. I have the same problem as described in this section and i understand that you have to get the original version. But where do you get the original version? And how do you know if it is the original version?

With high regards,

Reinyn

**Comment by mayonakasl** [comment]
it seems that you need all the dlcs to install

**Comment by MrAnderson001** [comment]
My fix was using the whole HF patcher after doing the DLC HF patch.

**Comment by k4yf0ur** [comment]
Consider following the [installation guide](https://docs.google.com/document/d/e/2PACX-1vS8Ap6CrmSNXRsKG9jsIMqHYuHM3Cfs5qE5nX6iIgfzLlcWnmiwzmOrp27ytEMX03lFNRR7U5UXJalA/pub) instead.
